rust/lance/src/index/vector/ivf.rs

```diff
@@ -714,6 +714,8 @@ pub async fn build_ivf_pq_index(
             vec![],
             None,
         )?;
+
+        info!("starting to compute partitions for PQ training, sample size");
```

Comment:
Did you mean to log the sample size or something?

Reply:
ah yes, fixing